composer.json

```diff
@@ -13,7 +13,8 @@
         "php": "^7.0",
         "ext-hash": "*",
         "ext-json": "*",
-        "ext-mongodb": "^1.8"
+        "ext-mongodb": "^1.8",
+        "jean85/pretty-package-versions": "^1.2"
```

Comment:
This provides a better interface around version strings returned by [ocramius/package-versions](https://packagist.org/packages/ocramius/package-versions). It is used to provide the current version of this library without having to maintain it in a version constant.